ui-tests/src/test/java/org/openmrs/reference/page/EditVitalsPage.java

```diff
@@ -0,0 +1,110 @@
+package org.openmrs.reference.page;
+
+import org.openmrs.uitestframework.page.Page;
```

Comment:
Change this to import from org.openmrs.contrib.qaframework.helper.page
```suggestion
import org.openmrs.contrib.qaframework.helper.page.```